learnArrays/learnArrays.js

```diff
@@ -77,6 +77,49 @@ export const addNum = arr => arr.map((num) => num + 2)
 //reduce method
 export const addAll = arr => arr.reduce((tot, num) => tot += num)
 
+// Find max and min numbers in the array
+
+export const findMaxAndMin = arr => {
+    let maxNum = Number.NEGATIVE_INFINITY;
+    let minNum = Number.POSITIVE_INFINITY;
+    arr.forEach(num => {
+        if (num > maxNum) {
+            maxNum = num
+        }
+        if (num < minNum) {
+            minNum = num
+        }
+    })
+    const finRes = [maxNum, minNum]
```

Comment:
Better to return max and min as object. something like `{max: x, min : y}`

Reply:
fixed